robocop/reports/robocop_version_report.py

```diff
@@ -13,9 +13,16 @@ class RobocopVersionReport(Report):
         Report generated by Robocop version: 2.0.2
     """
 
-    def __init__(self):
+    def __init__(self, compare_runs):
         self.name = "version"
         self.description = "Returns Robocop version"
+        super().__init__(compare_runs)
 
-    def get_report(self) -> str:
-        return f"\nReport generated by Robocop version: {__version__}"
+    def persist_result(self):
+        return {"generated_with": __version__}
+
+    def get_report(self, prev_results) -> str:  # TODO Add note about previous run version
```

Comment:
Is this comment needed here?

Reply:
It should be removed, it was my work in progress comment.